integration-cli/docker_api_stats.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/docker/docker/api/types"
+	"github.com/go-check/check"
+	"strings"
+	"time"
+)
+
+func (s *DockerSuite) TestCliStatsNoStreamGetCpu(c *check.C) {
+	dockerFile := fmt.Sprintf("FROM busybox\nRUN echo \"#/bin/sh\" > /bin/test.sh && echo \"while true;do\" >> /bin/test.sh && echo \"echo \"Hello world\"\" >> /bin/test.sh && echo \"done\" >> /bin/test.sh\nRUN chmod +x /bin/test.sh\nCMD /bin/test.sh")
```

Comment:
this will generate a Dockerfile like this 
<pre><code>FROM busybox
RUN echo "#/bin/sh" > /bin/test.sh && echo "while true;do" >> /bin/test.sh && echo "echo "Hello world"" >> /bin/test.sh && echo "done" >> /bin/test.sh
RUN chmod +x /bin/test.sh
CMD /bin/test.sh
</code></pre>
and the test.sh is like this
<pre><code>#/bin/sh
while true;do
echo Hello world
done
</code></pre>